docs/static/security/logstash.asciidoc

```diff
@@ -18,14 +18,10 @@ pipeline is halted if authentication fails.
 If encryption is enabled on the cluster, you also need to enable TLS/SSL in the
```

Comment:
This is no longer an "If" - it is now enabled by default. Therefore we should probably re-phrase "Starting in 8.0, security is enabled by default on the Elasticsearch cluster. Therefore you will need to enable TLS/SSL in the Elasticsearch output section of the Logstash configuration in order to allow Logstash to communicate with the Elasticsearch cluster.

Reply:
Note - using the word "encryption" is ambiguous - should refer to "security"